src/commands/guildLog.ts

```diff
@@ -0,0 +1,189 @@
+import { ApplicationCommandOptionType, EmbedBuilder } from "discord.js";
+import { Command } from "../interfaces/Command";
+import { BotEvents } from "mineflayer";
+
+export default {
+	data: {
+		name: "guildlog",
+		description: "Get the guild log!",
+		options: [
+			{
+				name: "member",
+				description: "What user do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+			{
+				name: "page",
+				description: "What page do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+		],
+	},
+	run: async (bot, interaction, args) => {
+		const member: string = args[0];
+		const page: number = args[1];
+
+		if ((member && page) == undefined) {
```

Comment:
Unsure if this is the intention, but this statement is not the same as `member == undefined && page == undefined`. Regardless, might be best to rewrite this with an or statement if that's what you're going for

Please also use triple equality (`===`) where possible.